docs/management/connectors/action-types/servicenow-itom.asciidoc

```diff
@@ -7,6 +7,11 @@
 
```

Comment:
At the end of the [Connector Configuration](https://kibana_117122.docs-preview.app.elstc.co/guide/en/kibana/master/servicenow-itom-action-type.html#servicenow-itom-connector-configuration) section there is the following:

> The ServiceNow user requires at minimum read, create, and update access to the Event table and read access to the sys_choice. If you don’t provide access to sys_choice, then the choices will not render.

I think it should be delete it now that we have the `Prerequisites` section.